README.md

```diff
@@ -27,6 +28,28 @@ Using Kubernetes primitives, administrators configure identities and bindings to
 * [What To Do Next?](#what-to-do-next)
 * [Code of Conduct](#code-of-conduct)
 
+## v1.6.0 Breaking Change
+
+With https://github.com/Azure/aad-pod-identity/pull/398, the [client-go](https://github.com/kubernetes/client-go) library is upgraded to v0.17.2, where CRD fields are now case sensitive. If you are upgrading MIC and NMI from v1.x.x to v1.6.0, no action is required since existing `AzureIdentity` and `AzureIdentityBinding` are backward-compatible.
```

Comment:
Also, one more note to add - For existing `AzureIdentity` and `AzureIdentityBinding`, MIC will convert all the **existing resources** on startup to the new format and on successfully doing that will write to `aad-pod-identity-config` configmap